test/integration/install/test_install.py

```diff
@@ -614,3 +615,43 @@ def test_install_tt_already_exists_with_symlink(tt_cmd, tmpdir):
     expected_bin = os.path.join(bin_path, "tt_v1.1.2")
     tarantool_bin = os.path.realpath(os.path.join(bin_path, "tt"))
     assert tarantool_bin == expected_bin
+
+
+@pytest.mark.parametrize("package_to_exclude, expected_rc", [
+    pytest.param(
+        "mage",
+        1,
+    ),
+    pytest.param(
+        "",
+        0,
```

Comment:
There are integration tests, which test `tt` installation - test/integration/install/test_install.py.

Reply:
Just deleted it. Also change test to check only excluded dependencies cause successful one we have in `test_tt_install` function.